tests/test_scanner.py

```diff
@@ -1,4 +1,3 @@
-import tempfile
```

Comment:
@hakancelik96 this import has **not** used, and it has **not** caught by `unimport`!!!

~> (~Removed manually~)

Reply:
because the tests contain the comment `# unimport: skip_file`

Reply:
Ok, I'm sorry.

Reply:
It’s not a big deal, it’s pretty normal to miss out on it.